src/libstd/sys/windows/c.rs

```diff
@@ -108,6 +107,9 @@ pub const SECURITY_SQOS_PRESENT: DWORD = 0x00100000;
 
 pub const FIONBIO: c_ulong = 0x8004667e;
 
+pub const STATUS_SUCCESS: NTSTATUS = 0x00000000;
+pub const STATUS_TIMEOUT: NTSTATUS = 0x00000102;
```

Comment:
Why did these get added?

Reply:
Because they are needed in `parking_lot/core/src/thread_parker/windows/keyed_event.rs`